db/schema_tables.cc

```diff
@@ -695,16 +695,16 @@ redact_columns_for_missing_features(mutation m, schema_features features) {
  * Read schema from system keyspace and calculate MD5 digest of every row, resulting digest
  * will be converted into UUID which would act as content-based version of the schema.
  */
-future<utils::UUID> calculate_schema_digest(distributed<service::storage_proxy>& proxy, schema_features features)
+future<utils::UUID> calculate_schema_digest(distributed<service::storage_proxy>& proxy, schema_features features, noncopyable_function<bool(std::string_view)> skip_predicate)
```

Comment:
Maybe the positive form would be nicer for understanding, include_predicate.

Reply:
Sure, both are equally fine with me, which makes the positive form win.

Reply:
One slight disadvantage of the positive form is that I can no longer pass a plain function pointer as a predicate, and instead have to negate it just to negate it back inside the `calculate_schema_digest` implementation. It's not a big issue though.